src/status_im/ui/screens/home/views/inner_item.cljs

```diff
@@ -13,60 +13,47 @@
             [status-im.ui.components.icons.vector-icons :as icons]
             [status-im.utils.contenthash :as contenthash]
             [status-im.utils.core :as utils]
-            [status-im.utils.datetime :as time])
-  (:require-macros [status-im.utils.views :refer [defview letsubs]]))
+            [status-im.utils.datetime :as time]
+            [status-im.utils.gfycat.core :as gfycat]))
 
-(defview mention-element [from]
-  (letsubs [contact-name [:contacts/contact-name-by-identity from]]
-    contact-name))
+(defn add-parsed-to-subheader [acc-subheader {:keys [type destination literal children]}]
+  (let [string (case type
+                 "paragraph"
+                 (str (reduce
+                       (fn [acc-string parsed-child]
+                         (add-parsed-to-subheader acc-string parsed-child))
+                       ""
+                       children) " ")
 
-(defn render-subheader-inline [acc {:keys [type destination literal children]}]
-  (case type
-    "paragraph"
-    (conj acc (reduce
-               (fn [acc e] (render-subheader-inline acc e))
-               [react/text-class " "]
-               children))
+                 "mention"
+                 (gfycat/generate-gfy literal)
 
-    "blockquote"
-    (conj acc (.substring literal 0 (dec (.-length literal))))
+                 "status-tag"
+                 (str "#" literal)
 
-    "codeblock"
-    (conj acc (.substring literal 0 (dec (.-length literal))))
+                 "link"
+                 destination
 
-    "mention"
-    (conj acc [react/text-class
-               [mention-element literal]])
```

Comment:
Here, if you just use this instead, and uncomment mention-element, what is the behavior?